The __remove_cv builtin only exists since g++ 16, so keep a fallback.

--- a/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/c++/v1/__type_traits/remove_cv.h
+++ b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/c++/v1/__type_traits/remove_cv.h
@@ -10,6 +10,8 @@
 #define _LIBCPP___TYPE_TRAITS_REMOVE_CV_H
 
 #include <__config>
+#include <__type_traits/remove_const.h>
+#include <__type_traits/remove_volatile.h>
 
 #if !defined(_LIBCPP_HAS_NO_PRAGMA_SYSTEM_HEADER)
 #  pragma GCC system_header
@@ -17,15 +19,20 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
+#if defined(_LIBCPP_COMPILER_GCC)
+template <class _Tp>
+struct _LIBCPP_NO_SPECIALIZATIONS remove_cv {
+  typedef __remove_volatile_t<__remove_const_t<_Tp> > type;
+};
+
+template <class _Tp>
+using __remove_cv_t _LIBCPP_NODEBUG = __remove_volatile_t<__remove_const_t<_Tp> >;
+#else
 template <class _Tp>
 struct _LIBCPP_NO_SPECIALIZATIONS remove_cv {
   using type _LIBCPP_NODEBUG = __remove_cv(_Tp);
 };
 
-#if defined(_LIBCPP_COMPILER_GCC)
-template <class _Tp>
-using __remove_cv_t _LIBCPP_NODEBUG = typename remove_cv<_Tp>::type;
-#else
 template <class _Tp>
 using __remove_cv_t _LIBCPP_NODEBUG = __remove_cv(_Tp);
 #endif
